gitversion.yml

```diff
@@ -1,33 +1,45 @@
 assembly-versioning-scheme: MajorMinorPatch
-mode: ContinuousDeployment
+mode: Mainline
 next-version: 1.29.0
```

Comment:
The doc says tagging (in git I assume) is the way to set the version number, but the gitversion.yml still has a next-version set. Is that normal?

Reply:
We don't want to use tags for versionning, because it's not "burned" in the source control itself, so it's not immutable.

Reply:
For tags, while that's true, it's unworkable for version detection unfortunately (because master can't detect what's been upgraded to auto-update its version number). We'll have to bite that bullet...

Reply:
As for the next-version, that's the base that is used for already published versions (the next stable version for Uno.Core is 1.29) It'll be ignored once tags are added.

Reply:
@carldebilly  But the doc actually state that we are planning to use them anyway ... which is acceptable as it's the standard way to do that in git / GitHub.

Reply:
Oups ... I lagged ...